tools/openms/OpenMSDatabasesInfo.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTDConverter.-->
+<!--Proposed Tool Section: [Utilities]-->
+<tool id="OpenMSDatabasesInfo" name="OpenMSDatabasesInfo" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@" profile="20.05">
+  <description>Prints the content of OpenMS' enzyme and modification databases to TSV</description>
+  <macros>
+    <token name="@EXECUTABLE@">OpenMSDatabasesInfo</token>
+    <import>macros.xml</import>
+    <import>macros_autotest.xml</import>
+    <import>macros_test.xml</import>
+  </macros>
+  <expand macro="requirements"/>
+  <expand macro="stdio"/>
+  <command detect_errors="exit_code"><![CDATA[@QUOTE_FOO@
+@EXT_FOO@
+#import re
+
+## Preprocessing
+mkdir enzymes_out &&
+mkdir mods_out &&
+
+## Main program call
+
+set -o pipefail &&
+@EXECUTABLE@ -write_ctd ./ &&
+python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hardcoded_json' &&
+@EXECUTABLE@ -ini @EXECUTABLE@.ctd
+-enzymes_out
+'enzymes_out/output.${gxy2omsext("tabular")}'
+-mods_out
+'mods_out/output.${gxy2omsext("tabular")}'
+
+## Postprocessing
+&& mv 'enzymes_out/output.${gxy2omsext("tabular")}' '$enzymes_out'
+&& mv 'mods_out/output.${gxy2omsext("tabular")}' '$mods_out'
+#if "ctd_out_FLAG" in $OPTIONAL_OUTPUTS
+  && mv '@EXECUTABLE@.ctd' '$ctd_out'
+#end if]]></command>
+  <configfiles>
+    <inputs name="args_json" data_style="paths"/>
+    <configfile name="hardcoded_json"><![CDATA[{"log": "log.txt", "threads": "\${GALAXY_SLOTS:-1}", "no_progress": true}]]></configfile>
+  </configfiles>
+  <inputs>
+    <expand macro="adv_opts_macro">
+      <param name="force" argument="-force" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Overrides tool-specific checks" help=""/>
+      <param name="test" argument="-test" type="hidden" optional="false" value="False" label="Enables the test mode (needed for internal use only)" help="">
+        <expand macro="list_string_san" name="test"/>
+      </param>
+    </expand>
+    <param name="OPTIONAL_OUTPUTS" type="select" optional="true" multiple="true" label="Optional outputs">
```

Comment:
that looks also like it is not needed

Reply:
Which leave the question where are the normal, non advanced params?

Reply:
The tool just produces two tabular output files (so no parameters) .. let me ask if they may be needed for downstream tools: 

allowed modifications:

```
FullId  FullName        Origin/AA       TerminusSpecificity     DiffMonoMass
Acetyl (T)      Acetylation     T       none    42.0106
Acetyl (Protein N-term) Acetylation     X       Protein N-term  42.0106
Acetyl (S)      Acetylation     S       none    42.0106
Acetyl (C)      Acetylation     C       none    42.0106
Acetyl (N-term) Acetylation     X       N-term  42.0106
Acetyl (K)      Acetylation     K       none    42.0106
Acetyl (Y)      Acetylation     Y       none    42.0106
Acetyl (H)      Acetylation     H       none    42.0106
Acetyl (R)      Acetylation     R       none    42.0106
Amidated (C-term)       Amidation       X       C-term  -0.984016
Amidated (Protein C-term)       Amidation       X       Protein C-term  -0.984016
Biotin (N-term) Biotinylation   X       N-term  226.078
Biotin (K)      Biotinylation   K       none    226.078
Carbamidomethyl (Y)     Iodoacetamide derivative        Y       none    57.0215
Carbamidomethyl (T)     Iodoacetamide derivative        T       none    57.0215
Carbamidomethyl (S)     Iodoacetamide derivative        S       none    57.0215
Carbamidomethyl (E)     Iodoacetamide derivative        E       none    57.0215
Carbamidomethyl (D)     Iodoacetamide derivative        D       none    57.0215
...
```

allowed enzymes:

```
OpenMS_AllowedEnzymes
no cleavage
unspecific cleavage
Chymotrypsin
Chymotrypsin/P
CNBr
Formic_acid
Lys-C
Lys-N
Asp-N
Asp-N/B
Asp-N_ambic
Arg-C
Arg-C/P
V8-DE
V8-E
leukocyte elastase
proline endopeptidase
glutamyl endopeptidase
Alpha-lytic protease
2-iodobenzoate
iodosobenzoate
staphylococcal protease/D
proline-endopeptidase/HKR
Glu-C+P
PepsinA + P
cyanogen-bromide
Clostripain/P
elastase-trypsin-chymotrypsin
Lys-C/P
PepsinA
TrypChymo
Trypsin/P
Trypsin
```